File: Tutorial 7/if_statements.py
print("Enter 3 numbers to see what is the largest")
num1 = (float(input("Input the first number: ")))
num2 = (float(input("Input the first number: ")))
num3 = (float(input("Input the first number: ")))
if (num1>=num2 and num1>=num3):
    print(str(num1) +" is the largest number")
elif (num2>=num1 and num2>=num3):
    print(str(num2) +" is the largest number")
else:
    print(str(num3) +" is the largest number")

File: C/tutorial_6/if_statements.c
/*********************************
Program: if_statements.c
Author: Jonathan Karr
This program introduces the if statement
print out the largest number
*********************************/

#include <stdio.h>

int main(){
    double num1, num2;
    printf("Enter 2 numbers to see which is bigger: ");
    scanf("%lf %lf", &num1, &num2);
    if(num1 > num2){
        printf("%g is the largest number\n", num1);
    }
    else if( num1 == num2 ){
        printf("The numbers are the same. They are both %g\n", num1);
    }
    else{
        printf("%g is the largest number\n", num2);
    }
    return 0;
}